rasa/core/nlg/template.py

```diff
@@ -109,14 +113,19 @@ def _fill_template(
             "text",
             "image",
             "custom",
-            "button",
+            "buttons",
             "attachment",
             "quick_replies",
         ]
         if template_vars:
             for key in keys_to_interpolate:
                 if key in template:
-                    template[key] = interpolate(template[key], template_vars)
+                    if key != "buttons":
+                        template[key] = interpolate(template[key], template_vars)
+                    else:
+                        template[key] = interpolate_buttons(
+                            template[key], template_vars
+                        )
```

Comment:
that's not needed, right?

Reply:
These are needed, the original code won't work otherwise.